src/app/DTT/check_box/index.js

```diff
@@ -0,0 +1,22 @@
+import { Checkbox, HStack } from '@chakra-ui/react'
+import { Read } from 'app/DTT/text'
+
+const Write = ({ questionCode, data, onSendAnswer }) => {
+  console.warn('data', data)
+  return (
+    <HStack spacing={2}>
+      <Checkbox
+        colorScheme="green"
+        isChecked={data?.value === 'true'}
+        value={`Yes`}
+        onChange={() => onSendAnswer('true')}
+      >{`Yes`}</Checkbox>
```

Comment:
Does it need a label? can we get the label to come from the backend

Reply:
I will need to ask the BE Team to send me the Label for the checkbox then.